Client/Frontend/Widgets/ToggleButton.swift

```diff
@@ -28,9 +28,9 @@ class ToggleButton: UIButton, Themeable {
         if selected {
             var rect = CGRect(size: bounds.size).inset(by: UX.Insets)
             // Fix crash for negative rect size
-            guard rect.height > 0, rect.width > 0 else { return }
```

Comment:
this is super obscure. I could reproduce this crash and it happened because `rect.height` and `rect.size.height` had different values. One was positive, the other negative 🙈 I haven't known this was possible 😲 

So it passed the guard here but later it crashed because of a negative height and thus a negative value for `corner` 🤦